choreolib/src/main/java/choreo/auto/AutoTrajectory.java

```diff
@@ -529,16 +553,12 @@ public Trigger atPose(Optional<Pose2d> pose, double toleranceMeters) {
    * @param toleranceMeters The tolerance in meters.
    * @return A trigger that is true when the robot is within toleranceMeters of the given pose.
    */
-  public Trigger atPose(Pose2d pose, double toleranceMeters) {
-    return atPose(Optional.of(pose), toleranceMeters);
-  }
-
-  private Supplier<Optional<Pose2d>> optionalFlipped(Optional<Pose2d> pose) {
-    return AllianceFlipUtil.optionalFlipped(pose, alliance, useAllianceFlipping);
+  public Trigger atTranslationFlipped(Pose2d pose, double toleranceMeters) {
```

Comment:
I think u should rename "atTranslationFlipped" to "atTranslation" and "atTranslation" to "atAbsoluteTranslation";
most of the time, people want the pose to be flipped.

For it's namesake, an atTranslation(Translation2d) overload should exist too.